App/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think you can add this to .gitignore, pretty sure it's just editor settings or something.

Reply:
That has been something I been wanting to ask you since target is bloating the PR and changing every time mvn runs, but I didn't know if it was something necessary. I will add it to git ignore and find a way to delete it from this PR entirely 🙂  

would this be a good approach to deleting those files/directory?  
https://docs.github.com/en/github/managing-large-files/removing-files-from-a-repositorys-history

Reply:
Yup, git rm --cached should do it! (For the record though, I _think_ that will still leave it in the history, despite what that article says. This is totally fine, though.